### PR TITLE
Uses defines for vending machine access

### PR DESCRIPTION
**Pull request by 81Denton**
Having to look up numbers in _DEFINES\access.dm was getting a little annoying.

### CODE REVIEW

File: code/modules/vending/youtool.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "Tools for tools."
 	icon_state = "tool"
 	icon_deny = "tool-deny"
-	//req_access_txt = "12" //Maintenance access
+	//req_access = list(ACCESS_MAINT_TUNNELS)
```
Comment (81Denton) [inline code/modules/vending/youtool.dm]: This access req was commented out to begin with. No idea why.

Comment (vuonojenmustaturska) [reply code/modules/vending/youtool.dm]: remove it

Comment (81Denton) [reply code/modules/vending/youtool.dm]: Remove the req or that it's commented out?

Comment (Cyberboss) [reply code/modules/vending/youtool.dm]: remove the enitre line

Comment (81Denton) [reply code/modules/vending/youtool.dm]: Gotcha

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**